servers/rendering/renderer_viewport.cpp

```diff
@@ -143,49 +150,49 @@ void RendererViewport::_configure_3d_render_buffers(Viewport *p_viewport) {
 				use_taa = false;
 			}
 
-			int width;
-			int height;
+			int target_width;
+			int target_height;
 			int render_width;
 			int render_height;
 
 			switch (scaling_3d_mode) {
 				case RS::VIEWPORT_SCALING_3D_MODE_BILINEAR:
 					// Clamp 3D rendering resolution to reasonable values supported on most hardware.
 					// This prevents freezing the engine or outright crashing on lower-end GPUs.
-					width = CLAMP(p_viewport->size.width * scaling_3d_scale, 1, 16384);
-					height = CLAMP(p_viewport->size.height * scaling_3d_scale, 1, 16384);
-					render_width = width;
-					render_height = height;
+					target_width = p_viewport->size.width;
```

Comment:
Also fixed this, this has been a long time beef of mine. The target size was set to the render size even when scaling was applied as a workaround when we didn't tell the renderer what scaling was being applied.

This code should have been fixed up when we fixed that issue. Doing it now seeing we're cleaning this stuff up anyway.